src/services/PartsKit.php

```diff
@@ -94,6 +95,55 @@ public static function getNav(): ?array
         return $items;
     }
 
+    public static function getTheme()
+    {
+        $themes = [
+            'light' => [
+                'background' => '#ededed',
+                'main-background' => '#fff',
+                'text' => '#202020',
+                'nav-icon' => '#148bbe',
+                'nav-item-text-hover' => '#202020',
+                'nav-item-background-hover' => '#dbdbdb',
+                'nav-subitem-text-hover' => '#202020',
+                'nav-subitem-background-hover' => '#dbdbdb',
+                'nav-subitem-background-active' => '#148bbe',
+                'nav-subitem-text-active' => '#fff',
+                'controls-text' => '#a7a9ac',
+                'controls-border' => '#dbdbdb',
+            ],
+
+            'dark' => [
+                'background' => '#2f2f2f',
+                'main-background' => '#333',
+                'text' => 'rgba(255, 255, 255, 0.8)',
+                'nav-icon' => '#1ea7fd',
+                'nav-item-text-hover' => '#fff',
+                'nav-item-background-hover' => 'rgba(250, 250, 252, 0.1)',
+                'nav-subitem-text-hover' => '#fff',
+                'nav-subitem-background-hover' => 'rgba(250, 250, 252, 0.1)',
+                'nav-subitem-background-active' => '#1ea7fd',
+                'nav-subitem-text-active' => '#fff',
+                'controls-text' => '#999',
+                'controls-border' => 'rgba(255, 255, 255, 0.1)',
+            ],
+        ];
+
+        $theme = self::getConfig('theme');
```

Comment:
Maybe move the `?? light` to here

Reply:
hmm from 137? that's a lil diff. no need to default here since the config itself has the defaults. the check on 137 protects against passing an invalid string

Reply:
Ah right ok, fair.